JDI.Light/JDI.Light/Elements/Complex/Table/Table.cs

```diff
@@ -3,27 +3,46 @@
 using System.Linq;
 using JDI.Light.Asserts;
 using JDI.Light.Elements.Base;
+using JDI.Light.Extensions;
 using JDI.Light.Factories;
+using JDI.Light.Matchers;
 using OpenQA.Selenium;
+using static System.Globalization.CultureInfo;
+using static System.String;
 
 namespace JDI.Light.Elements.Complex.Table
 {
     public class Table : UIElement
     {
+        public static Func<string, string> Simplify { get; set; } = s => s.ToLower(InvariantCulture).Replace("[^a-zA-Z0-9]", "");
+        public static Func<string, string> TrimPreview { get; set; } = el => el.Trim().Replace(" +", " ").Replace("\r\n", "");
```

Comment:
Please do not create a static members inside a class that can have several instances.
Different tables should be able to have different implementations of this property.